cli/medperf/entities/schemas.py

```diff
@@ -105,3 +106,6 @@ def default_status(cls, v):
         if v is not None:
             status = Status(v)
         return status
+
+class DeployableEntity(DeployableSchema, Entity):
+    pass
```

Comment:
This "implementation" is necessary at two places:
1. Entity list command - to declare that applicable objects should be both DeployableSchema and Entity.
2. for Entity list test purposes
However, placing implementation here breaks code design a bit: it make `schemas.py` dependable from `interface.py`. Maybe it's worth to move this class definition to other place?

Reply:
I see your point. Maybe this could live in the interface file instead? I also think that, just for clarity, entities that apply to this description should inherit from this DeployableEntity class.